pkg/codegen/docs/constructor_syntax_generator_test.go

```diff
@@ -51,6 +62,11 @@ func TestConstructorSyntaxGeneratorForSchema(t *testing.T) {
 							Type: "boolean",
 						},
 					},
+					"fooNumericEnum": {
+						TypeSpec: schema.TypeSpec{
+							Ref: "#/types/test:index:ExampleEnum",
+						},
+					},
```

Comment:
Should we also add a test case for non-numeric enum?

Reply:
Done ✅